packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -1072,15 +1155,161 @@ test('lib-wallet addOffer for autoswap swap', async t => {
   );
 });
 
+test('lib-wallet can give attestations in offers', async t => {
+  const {
+    wallet,
+    attestationPublicFacet,
+    attestationBrand,
+    attestationIssuer,
+  } = await setupTest({ attestation: true });
+
+  const issuerManager = wallet.getIssuerManager();
+  await issuerManager.add('Attestation', attestationIssuer);
+  await wallet.makeEmptyPurse('Attestation', 'staking purse');
+
+  const invitation = await attestationPublicFacet.makeReturnAttInvitation();
+
+  const rawId = '1593482020370';
```

Comment:
Is that an arbitrary Id? or something carefully calculated to get expected results?
I changed it and the test still works, so it seems to be arbitrary. I suggest:

```suggestion
  const rawId = '123-arbitrary';
```

Reply:
Done